### ISSUE TITLE
How do we work with this folder? 

### ISSUE THREAD

**Issue by sarambl** [NordicESMhub/escience-2022#1]
Hi! Maybe I totally missed some planning here, but is there a plan to the workflow of the course? 
Do we add everyone as collaborators? Do they all make their own branch or rather their own fork? Will anyone be main responsible for maintaining the repo?

**Comment by sarambl** [comment]
Also, how does this repo and the tutorial one from pangeo-data interact? Will that one just stay as is with the tutorial material, and then we add our own material to this repo? If so, I suggest a structure along the lines of last year. 
![image](https://user-images.githubusercontent.com/17406708/198548228-6142df7b-754d-4c90-85da-5288f814dd86.png)

**Comment by MichaelSchulzMETNO** [comment]
yes - lets do as last year - can you prepare this in this repo @sarambl ?

**Comment by sarambl** [comment]
I can try, but can't promise I'll have time. I'm a bit over-worked! 
We might want to copy something like this: https://nordicesmhub.github.io/forces-2021/setup/organize.html?highlight=github

In general I think it's not a bad practice if everyone makes their own fork, but maybe you have an opinion on best practice here @annefou ? I don't quite remember what we did last year, but I think we might have had a branch per group or something?

**Comment by adagj** [comment]
But if everyone only work on their own fork, it is difficult to share and also look for inspiration among the students... we can always have a different branch for each group and commit from the forks to that particular branch and it that way share. If the students are not too familiar with git it might be messy... But you have more experience with this than me. 

Do I need to remove my plankton folder? I can do so :-)

**Comment by sarambl** [comment]
So, the idea would be to regularly pull from the main repo and also to submit pull requests.. Much like with a branch, but I see how people might forget to do that!

**Comment by sarambl** [comment]
@adagj , no need to remove anything, I think whoever does a tidying can just move it to the right place.

@MichaelSchulzMETNO Honestly I would prefer if someone else does the cleaning and set up, because I am really at max capacity already for a while!

**Comment by MichaelSchulzMETNO** [comment]
ok, I can do that over the weekend. And suggest a way to git-work together.

**Comment by annefou** [comment]
We usually advice not to use this repo and use the organisation eScience-course and the template we prepare. I suggest one repo per group as one group should use similar datasets. This will make their workflows much simpler. Each group mentor needs to admin his/her own repo and add other team members as collaborators of their group repo.


You can add examples to GitHub.com/pangeo-data/escience-2022 (pull request).

**Comment by MichaelSchulzMETNO** [comment]
Honestly, I prefer one repo for the course. In addition to the pangeo repo. It worked well in the last years. And we want to foster exchange in the whole course group. There is also data use overlap.

**Comment by MichaelSchulzMETNO** [comment]
Of course, if all group repos are public, exchange is possible, I am not dogmatic on this.

**Comment by adagj** [comment]
Several groups work on quite similar subjects and will use the same data and exchange methods and scripts (I hope), so I also prefer one repro. The scripts I have uploaded so far can be useful for all groups I think, and I'm afraid it will be confusing to copy and update among many repos if we have one for each group. But I don't have very strong opinions on this, just expect the students to be easily git confused :-)

> We usually advice not to use this repo and use the organisation eScience-course and the template we prepare. I suggest one repo per group as one group should use similar datasets. This will make their workflows much simpler. Each group mentor needs to admin his/her own repo and add other team members as collaborators of their group repo.
> 
> You can add examples to GitHub.com/pangeo-data/escience-2022 (pull request).

@annefou Which repo is "this repo"? (sorry for being slow..)

**Comment by annefou** [comment]
This repo ==  https://github.com/NordicESMhub/escience-2022

I think you should do what you want. 
My experience from the previous years is that students are unable to learn and use GitHub properly during the course. We usually need to clean their mess on a regular basis (which I won't do this year).  Having their own repo and/or per group is much easier and can give them a bit more confidence and this is anyway what they usually do during their day to day work.

Examples could be added to GitHub.com/pangeo-data/escience-2022 (I could add mentors, etc. as collaborators to the repo).

**Comment by annefou** [comment]
> Of course, if all group repos are public, exchange is possible, I am not dogmatic on this.

Having private repos would certainly not help students to learn and understand Open Science practices...

**Comment by MichaelSchulzMETNO** [comment]
Ok suggestion:

Lets try to do one public repository per mentor-group this year. I made a test and created a https://github.com/eScience-course/Tjaernoe2022, which is mainly for my test purposes. Should be easy to do for all assistants. I chose the MIT licence. 

Examples will be visible to all if repos are public

Suggested group name by number: https://github.com/eScience-course/Tjaernoe2022-group1
See google drive escience-2022 participants  for group names and github names.

Private slides and presentations : we could store in the course google drive for all participants.
